content/introduction-and-setup/walkthrough/creating-a-project/index.md

```diff
@@ -0,0 +1,85 @@
+   1. **Mac Users**: Pick the options for “Use git for version control.” and “Create a .gitignore file to ignore inessential files”.
+1. Visual Studio will now create a **Solution** to hold your console **Project**. The project is your current app. The project contains all of the files that app needs to build and run. The solution is a workspace that combines multiple projects, which are usually related to each other. The solution file type is **.sln**. Once created, Visual Studio opens a new project window that displays your `Program.cs` file.
+1. To see the other files inside this solution, you will need to view either the Solution (Mac Users) or the Solution Explorer (Windows Users). You’ll see the project file tree containing a file called `Program.cs` in a pane called Solution Explorer.
+1. You are new to C# and we’ll go over the syntax present in Program.cs in time. For now, can you guess what line 2 accomplishes?
+
+   {{< highlight csharp "linenos=true" >}}
+      // See https://aka.ms/new-console-template for more information
+      Console.WriteLine("Hello, World!");
+   {{< / highlight >}}
+  
+   Click on the *Run* button (a triangle button located above the `Program.cs` panel) indicated in one of the images above to run the project and see the output.
+
+1. A console window should pop up with the line “Hello World” printed. That’s it. You have created and executed your first C# application!
+
+{{% notice blue "Tip" "rocket" %}}
+The first time you run a console app in Visual Studio, you may be prompted to allow VS to access the terminal. This is ok.
+This may also take longer than a few seconds to run the very first time.
+{{% /notice %}}
+
+## Troubleshooting
+
+This app is printing to the terminal. If you are not able to see the output, look inside the project’s terminal.
+
+If you would like more instructions on creating and running this project check out the following documentation:
+
+[Windows Users](https://learn.microsoft.com/en-us/visualstudio/get-started/csharp/visual-studio-ide?view=vs-2022)
+[Mac Users](https://learn.microsoft.com/en-us/visualstudio/mac/ide-tour?view=vsmac-2022)
+
+### Hello, Solution!
+
+You’ve just created your first C# project. Congrats! In fact, you’ve also just created your own C# solution. Your `HelloWorld` project is nested within a solution called `HelloWorld`. A solution behaves like a container for related projects and other Visual Studio settings.
+
+A C# project contains all the code to run a particular application. Along with the `Program.cs` file you ran just a moment ago, you may have also noticed a `Dependencies` folder. Many applications require extra code like dependencies or other compiling configurations to execute.
+
+You can create another project inside of the `HelloWorld` solution very easily. Right click on the solution name to add a new project, another console app as above, and name it `Hello<YourName>`. Change the starter code in `Program.cs` to greet you by name.
```

Comment:
HelloWorld(main) solution right click (for mac)

Reply:
I am confused. This is saying the same thing as "Right click on the solution name"

Reply:
Technically yes, Up until your reply I did not realize that the (main)- was referring to the current branch.  Might be a nice note as students are just starting out on their journey.

Reply:
Added a tip above this to make sure students don't get confused.

Reply:
Thanks for adding the tip Sally!